app/addons/cors/components/OriginTable.js

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import OriginRow from "./OriginRow";
+
+export default class OriginTable extends Component {
+
+  constructor (props) {
+    super(props);
+  }
+
+  createRows () {
+    return _.map(this.props.origins, function (origin, i) {
+      return <OriginRow
+        updateOrigin={this.props.updateOrigin}
+        deleteOrigin={this.props.deleteOrigin}
+        key={i} origin={origin} />;
+    }, this);
+  }
+
+  render () {
+    if (!this.props.origins) {
```

Comment:
Could you do `const {origins, isVisible} = this.props;` at the beginning of render.